watertap/core/wt_database.py

```diff
@@ -44,7 +47,8 @@ def __init__(self, dbpath=None):
                 "techno_economic",
             )
         else:
-            self._dbpath = dbpath
+            self._dbpath = os.path.dirname(dbpath)
+            self._filename = os.path.basename(dbpath).split(".")[0]
```

Comment:
I did not use `pathlib` because `os` was already used in this file. Is there any reason why `os` was used instead of `pathlib` for this class?

Reply:
Are there other areas of watertap that would expect an `os` path instead of a `pathlib` path?

Reply:
> I did not use `pathlib` because `os` was already used in this file. Is there any reason why `os` was used instead of `pathlib` for this class?

I don't think so. We should switch to pathlib where/when convenient.

Reply:
Ok, I replaced os with pathlib.